packages/slate/src/interfaces/path.ts

```diff
@@ -118,6 +118,14 @@ export const Path = {
     )
   },
 
+  /**
+   * Check if the path of previous sibling node exists
+   */
+
+  hasPrevious(path: Path): boolean {
+    return path.length !== 0 && path[path.length - 1] > 0
```

Comment:
Do you need the first part of this check?  Otherwise, looks good!

Reply:
Fixed